support-frontend/assets/pages/[countryGroupId]/components/checkoutComponent.tsx

```diff
@@ -709,14 +745,24 @@ export function CheckoutComponent({
 			};
 			setErrorMessage(undefined);
 			setErrorContext(undefined);
-			const createResponse = await fetch('/subscribe/create', {
+
+			const fetchCreateSupportWorker = fetch('/subscribe/create', {
```

Comment:
I _think_ this should be `() => fetch('/subscribe/create', {`, i.e. add an extra layer of function application. Otherwise we'll be retrying the exact same promise each time (which I don't think will make a difference, since a promise can only have one outcome).

Reply:
Think we now only need the same promise here. After this, the standalone processPaymentWithRetries mechanism will handle the 10 pending retries.